README.md

```diff
@@ -95,13 +95,60 @@ If the above doesn't fix the problem, open your `.bash_profile` and add the foll
 
 #### Ansible
   You can use a task:
-
-```
-- name: nvm
-  shell: >
-    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
-  args:
-    creates: "{{ ansible_env.HOME }}/.nvm/nvm.sh"
+  
+```yml
+- hosts: webservers
```

Comment:
this is a lot of content to add to the readme; maybe we could wrap it in a `<details>`?

Reply:
yeah sorry - actually - this will fail without a defaults/main.yml file 
```yml
---
nvm:
  user: '{{ ansible_user_id }}'
  version: v0.34.0
  node_version: '6.9.0'
 
```
fatal: [13.238.201.177]: FAILED! => {"msg": "The field 'become_user' has an invalid value, which includes an undefined variable. The error was: 'nvm' is undefined"}

there's a bunch of plumbing here necessary - ideally we'd point user to this repo 
https://github.com/leonidas/ansible-nvm
but it's way out of date. fyi - @leonidas